src/components/generic/movies/movies.tsx

```diff
@@ -43,23 +43,28 @@ const RenderMovies = (movies: Movie[], loader: boolean) => {
   return (
     <>
       {movies !== undefined &&
-        movies.map(({poster_med, rating, year, genres, title, imdb}, key) => (
-          <ModalProvider.Consumer key={key}>
-            {({handleOpen}) => {
-              return (
-                <MovieCard
-                  title={title}
-                  imdb={imdb}
-                  genres={genres}
-                  posterMed={poster_med}
-                  rating={rating}
-                  year={year}
-                  handleOpen={handleOpen}
-                />
-              )
-            }}
-          </ModalProvider.Consumer>
-        ))}
+        movies.map(
+          (
+            {poster_path, vote_average, release_date, genres, title, imdb},
+            key,
+          ) => (
+            <ModalProvider.Consumer key={key}>
```

Comment:
Future consideration: use hooks instead of this pattern.

Reply:
I saw the pattern of `useCONTEXT_NAME` in Epic React I will definitly use it